utils/codegen/asn1-to-json/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Jinja2==3.0.3
+asn1tools==0.1.1
```

Comment:
I got this error trying to install requirements:
ERROR: No matching distribution found for asn1tools==0.1.1